vhdbuilder/packer/list-images.ps1

```diff
@@ -34,7 +34,7 @@ foreach($image in $imageList) {
         $isExist=$false
         foreach($bom in $bomList) {
             if ($bom.id -eq $id) {
-                $bom.repoDigests = $repoDigests
+                $bom.repoDigests = @($repoDigests)
```

Comment:
```
            if (-not $isExist) {
                $bom=[pscustomobject]@{
                    id=$id;
                    repoTags=@($repoTags);
                    repoDigests=@();
                }
                $bomList+=$bom
            }
```
repoDigests is assigned to am empty array so I think that we can just append the new value to it.

Reply:
Thanks for the advice. Done.